lib/evss/auth_headers.rb

```diff
@@ -49,6 +49,11 @@ def eauth_json
       }.to_json
     end
 
+    def iso8601(date)
+      return nil unless date
+      date.iso8601
```

Comment:
the iso8601_birth_date method uses the `&.` notation.  That could be used here instead of the nil check

Reply:
@erluti I'm new to ruby, so I'm sure I'm missing something, but I don't see how `&.` would allow to avoid the nil check?

Reply:
yes!
`date&.iso8601` will do the iso call unless date is nil, then it will just return nil.
Rails has a method to do something similar called `try`.  `date.try(:iso8601)`

Reply:
I would tend to write it as `date.iso8601 if date` because conditionals that return false without an else return nil

Reply:
Ah, you can use `&.` on a method! I do like the look of `date.iso8601 if date` better. Thanks!

Reply:
Wait, I'm gonna get rid of the separate function all together and just use the `&` to the existing code. Nothing like a one character fix.